packages/apps/marmalade-website/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@kadena/marmalade-website",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "test": "echo \"No tests yet.\""
```

Comment:
We have some kadena specific package json configurations. We used to be able to look at `lib/bootstrap-lib`, but I'm not sure if that's up to date. @alber70g Would you still recommend referencing that app for project setup?